zerver/models.py

```diff
@@ -165,6 +165,7 @@ class Realm(ModelReprMixin, models.Model):
 
     date_created = models.DateTimeField(default=timezone_now)  # type: datetime.datetime
     notifications_stream = models.ForeignKey('Stream', related_name='+', null=True, blank=True, on_delete=CASCADE)  # type: Optional[Stream]
+    signups_stream = models.ForeignKey('Stream', related_name='+', null=True, blank=True, on_delete=CASCADE)  # type: Optional[Stream]
```

Comment:
let's call this user_signup_notification_stream?
And in this commit maybe also change the one above to stream_creation_notification_stream.

Reply:
from https://chat.zulip.org/#narrow/stream/design/topic/notification.20streams, we're just changing the new one, to signup_notifications_stream.